config/software/cacerts.rb

```diff
@@ -19,16 +19,17 @@
 license "MPL-2.0"
 license_file "https://github.com/bagder/ca-bundle/blob/master/README.md"
 
-default_version "2016.01.20"
+default_version "2016-04-20"
 
-version "2016.01.20" do
-  source md5: "36eee0e80373937dd90a9a334ae42817"
-  source url: "https://raw.githubusercontent.com/bagder/ca-bundle/dfcc02c918b7bf40ed3a7f27a634c74ef4e80829/ca-bundle.crt"
+source url: "https://curl.haxx.se/ca/cacert-#{version}.pem"
+
+version "2016-04-20" do
+  source md5: "782dcde8f5d53b1b9e888fdf113c42b9"
```

Comment:
While we are at it can we include the sha256 as well since omnibus supports that now?